openshift4/templates/docgen.dc.yaml

```diff
@@ -120,7 +120,7 @@ objects:
               livenessProbe:
                 failureThreshold: 3
                 httpGet:
-                  path: /health
+                  path: api/v2/health
```

Comment:
I think there needs to be a leading slash for both the paths

Reply:
thanks Ben!!